lib/anycable/rpc_handler.rb

```diff
@@ -106,6 +108,7 @@ def rack_env(request)
         "PATH_INFO" => uri.path,
         "SERVER_PORT" => uri.port.to_s,
         "HTTP_HOST" => uri.host,
+        REMOTE_ADDR => request.headers[REMOTE_ADDR],
```

Comment:
Let's simplify this and remove both the constant and the check in the `build_headers` method:

```suggestion
        "REMOTE_ADDR" => request.headers.delete("REMOTE_ADDR"),
```

Reply:
Check in `build_headers` was intentional because according to the current precedence it's invoked before the element deletion. I'll fix that. Take a look if you like it more.

Reply:
Hm, what about a more descriptive, step-by-step, way

```ruby
{
   "REQUEST_METHOD" => "GET",
    ...
}.tap do |env|
  env["REMOTE_ADDR"] = headers.delete("REMOTE_ADDR")
  env.merge!(build_headers(headers))
end
```

Reply:
I made a try without `tap`